build.gradle

```diff
@@ -9,11 +9,11 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.0'
```

Comment:
Please revert all changes to this file

Reply:
ok

Reply:
This file still needs its changes reverted.

Reply:
Just done that. Reverted to original gradle file